setup.py

```diff
@@ -79,7 +79,8 @@ def _maintain_symlinks(symlink_type, base_path):
             if 'ansible-playbook' in symlink_data['script']['ansible']:
                 _cache_symlinks(symlink_data)
             else:
-                raise
+                raise RuntimeError("Symlinks in ./bin appear "
+                                   "to be missing or broken")
```

Comment:
Maybe: "Pregenerated symlink list was not present and expected symlinks in ./bin were missing or broken.  Perhaps this isn't a git checkout?"

Reply:
I'm happy for that to go in, only thought is that the original issue (for me at least) was related to a git checkout on windows, so the final sentence might be a bit of red herring. But overall I think this is a great improvement.

Reply:
Okay, let's do this.